PolicyAdministrationPoint/src/main/java/it/cnr/iit/ucs/pap/PolicyAdministrationPoint.java

```diff
@@ -85,8 +86,10 @@ public String retrievePolicy( String policyId ) {
     @Override
     public boolean addPolicy( String policy ) {
         Reject.ifBlank( policy );
-        PolicyWrapper policyWrapper = PolicyWrapper.build( policy );
-        if( policyWrapper == null ) {
+        PolicyWrapper policyWrapper;
```

Comment:
formatting! the line in 89 onwards should be indented inside the preceeding if

Reply:
I don't see the problem here. could you explain it again?

Reply:
ah sorry my mistake. The line above was deleted

Reply:
no problem